### ISSUE TITLE
Full screen mode X RetroArch

### ISSUE THREAD

**Issue by AlanRibas** [libretro/RetroArch#12302]
I don't know if RetroArch was actually programmed to do this or if it is a Bug.
I recently chose to leave the RetroArch window in full screen mode.
And most of the time it stays that way.
However, when I click on the Main Menu \ Show Desktop Menu option, the Retroarch window that was in full screen mode is minimized, showing then the Desktop menu.
When I close this Desktop window, Retroarch still remains in Window mode.
I leave RetroArch and when I return the window mode is still there.
Basically, clicking on the "show Desktop Menu" option resets my setup beforehand. Is there a way to fix this?

**Comment by LibretroAdmin** [comment]
What OS/platform?

**Comment by AlanRibas** [comment]
> What OS/platform?

Windows 10 PRO.
